packages/web/src/config.ts

```diff
@@ -1,11 +1,12 @@
 // RWJS_ENV and RWJS_DEBUG_ENV
-// are defined in Webpack.common.js and Vite.config.js
+// are defined in Vite.config.js
 
 // Note: These lines are useful during RSC/SSR development but will execute for all projects, even those without RSC/SSR
 // console.log('config.ts')
 // console.log('config.ts', RWJS_ENV)
 // console.log('config.ts')
 
+// TODO(jgmw): Ask for clarification on the following
```

Comment:
WE can just remove the comments.

If we want we can now use `globalThis.RWJS_ENV.RWJS_*` if we want